src/tests/ftest/nvme/io.yaml

```
@@ -40,15 +40,15 @@ ior:
   - "RP_4G1"
  dfs_destroy: False
  ior_sequence:
-#   - [scmsize, nvmesize, stripesize, blocksize, clientslots]
-    - [6000000000, 214748364800, 1048576, 17179869184, 1]        #[6G, 200G, 1M, 16G, 1]
-    - [1000000000, 9000000000, 4096, 1073741824, 1]              #[1G, 9G, 4k, 1G, 1]
-    - [10000000000, 9000000000, 2048, 1073741824, 1]             #[10G, 9G, 2k, 1G, 1]
-    - [3000000000, 9000000000, 8, 10485760, 1]                   #[3G, 9G, 8B, 10M, 1]
-    - [4000000000, 9000000000, 24, 25165824, 1]                  #[4G, 9G, 24B, 24M, 1]
-    - [1000000000, 9000000000, 2056, 4227136, 1]                 #[1G, 9G, 2056, 4M, 1]
-    - [6000000000, 322122547200, 134217728, 34359738368, 1]      #[6G, 300G, 128M, 32G, 1]
-    - [10000000000, 64424509440, 32, 44040192, 1]                #[10G, 60G, 32B, 42M, 1]
-    - [62277025792, 1000204886016, 2048, 1073741824, 2]          #[58G, 900G, 2k, 1G, 2]
-    - [62277025792, 1000204886016, 8, 10485760, 2]               #[58G, 900G, 8B, 10M, 2]
-    - [62277025792, 1000204886016, 33554432, 23622320128, 5]     #[58G, 900G, 32M, 22G, 5]
+#   - [scmsize%, nvmesize%, stripesize, blocksize, clientslots]
+    - ["2%", "75%", 1048576, 17179869184, 1]        #[2%, 75%, 1M, 16G, 1]
+    - ["1%", "4%", 4096, 1073741824, 1]             #[1G, 9G, 4k, 1G, 1]
```

```suggestion
    - ["1%", "4%", 4096, 1073741824, 1]             #[1%, 4%, 4k, 1G, 1]
```
Don't you want to update the rest of the comments?

---

Good catch. Updated now.